tutorials/gas_assembly/assembly.py

```diff
@@ -190,8 +190,7 @@ def assembly(n_ax_zones, n_inactive, n_active, add_entropy_mesh=False):
 
     # reflector is 40 percent helium by volume (arbitrary assumption), with helium
     # at the inlet conditions
-    rho = coolant_density(specs.inlet_T)
-    matrix_density = 1700
```

Comment:
was this line not being used? for `matrix_density`?

Reply:
Line 194 just redeclares the `matrix_density = 1700` that is already declared in line 119 to the same value. The code works the same without line 194. (I added a units comment to 193 as well just for extra clarity)